Android/PowerUp/src/com/example/powerup/MainActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.example.powerup;
+
+import com.example.powerup.R;
+
+import android.app.ListActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+public class MainActivity extends ListActivity {
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		Intent myIntent = new Intent(MainActivity.this, Game.class);
```

Comment:
Why are you using this approach instead of just launching the Game activity directly?